Doc/whatsnew/3.10.rst

```diff
@@ -229,7 +229,7 @@ PEP 634: Structural Pattern Matching
 ------------------------------------
 
 Structural pattern matching has been added in the form of a *match statement*,
-a data object (sequences, mappings, primitive data types as well as class
+a data object (which consists of sequences, mappings, primitive data types as well as class
 instances) and *case statements* of patterns with associated actions. Pattern
```

Comment:
I still think this is confusing -- to me it suggests that "a data object" was added to the language. Does this refer to the (syntactic) concept of a *pattern*? How about the following rewrite?
```suggestion
and *case statements* of patterns with associated actions. Patterns 
consists of sequences, mappings, primitive data types as well as class instances. Pattern
```

Reply:
Good point re: addition. Applied your suggestion.